src/userSync.js

```diff
@@ -0,0 +1,175 @@
+import * as utils from 'src/utils';
+
+export function newUserSync() {
```

Comment:
This follows the pattern in `config` (https://github.com/prebid/Prebid.js/blob/master/src/config.js#L28). It allows for resetting the state of this module in between tests.